extensions/ql-vscode/test/vscode-tests/minimal-workspace/local-queries/local-databases.test.ts

```diff
@@ -765,7 +782,7 @@ describe("local databases", () => {
 
     describe("when codeQL.codespacesTemplate is set to false", () => {
       it("should not create a skeleton QL pack", async () => {
-        jest.spyOn(Setting.prototype, "getValue").mockReturnValue(false);
+        jest.spyOn(config.Setting.prototype, "getValue").mockReturnValue(false);
```

Comment:
These other tests could do with some tidying (e.g. by mocking the specific setting, instead of the general `getValue` function 🤔), but I can do that in a separate PR!